src/components/Layout/MenuStyles.js

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components'
+
+import { background, blue, pink } from '@utils/theme'
+
+export const Logo = styled.img`
+	padding: 6px;
+`
+
+export const MobileNav = styled.div`
+	background: ${background};
+	bottom: 0;
+	box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+	display: flex;
+	flex-wrap: nowrap;
+	font-size: 12px;
+	justify-content: space-around;
+	line-height: 42px;
+	position: fixed;
+	width: 100%;
+
+	a {
+		padding: 0 6px;
+
+		&:after {
+			top: 0;
+			left: 6px;
+			right: 6px;
+		}
+	}
+`
+
+export const Space = styled.span`
+	flex-grow: 1;
+`
+
+export const Wrapper = styled.menu`
+	background: ${background};
+	display: flex;
+	height: 64px;
+	padding: 0 calc((100vw - 1440px) / 2);
+
+	.headroom--scrolled & {
+		box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+	}
+
+	a {
+		align-items: center;
+		color: ${pink};
+		display: flex;
+		position: relative;
+		text-decoration: none !important;
+		white-space: nowrap;
+	}
+
+	a:first-child {
+		flex-shrink: 0;
+	}
+
+	a {
+		&:after {
+			background: ${blue};
+			content: '';
+			display: block;
+			height: 0;
+			position: absolute;
+			transition: all 0.3s ease;
+		}
+
+		&[aria-current='page']:after,
```

Comment:
Should also get this to apply to subpages - e.g. the `transcripts` page should have the active state when viewing a `/transcripts/whatever/` subpage...